lib/sink.js

```diff
@@ -8,52 +8,57 @@ const path = require('path');
 const fs = require('fs');
 
 
-const SinkFs = module.exports = function (fileDir) {
-    if (!(this instanceof SinkFs)) { return new SinkFs(fileDir); }
-    assert(fileDir, '"fileDir" must be provided');
+module.exports = class SinkFs {
+    constructor (fileDir) {
+        assert(fileDir, '"fileDir" is missing');
```

Comment:
It doesn't make sense to not pass an absolute path, as the rename you do later will either put things into node_modules, or some other place. Enforcing absolute path with a nice error makes sense, IMO. Not a biggie, though 😄